api/Pipfile.lock

```diff
@@ -0,0 +1,179 @@
+{
```

Comment:
Same comment as the Pipfile. (Consider moving to "semester-project-team-2" directory).